### PR TITLE
Update to latest gradle 6 version (6.9.2)

### PR DESCRIPTION
**Pull request by dbast**
Done via: ./gradlew wrapper --gradle-version 6.9.2

Does not depend on old log4j version anymore:
https://blog.gradle.org/log4j-vulnerability

### CODE REVIEW

File: gradlew.bat [hunk 1]
```diff
@@ -1,100 +1,89 @@
...
-:win9xME_args
-@rem Slurp the command line arguments.
-set CMD_LINE_ARGS=
-set _SKIP=2
-
-:win9xME_args_slurp
-if "x%~1" == "x" goto execute
-
-set CMD_LINE_ARGS=%*
-
```
Comment (Lojjs) [inline gradlew.bat:78]: I'm not 100 % sure how this is used, but you PR seems to remove it, which I would advise against unless you have strong reasons for it

Comment (dbast) [reply gradlew.bat:78]: same here this file belong 100% to the gradle wrapper and is updated when calling the mentioned update command: `./gradlew wrapper --gradle-version 6.9.2`.

**Comment by dbast** [comment]
To make it super clear: the PR only contains content that was created by the single command `./gradlew wrapper --gradle-version 6.9.2`... nothing was manually changed/added/removed.

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9.2-bin.zip
```
[the remark this answers is not in the document]
Comment (dbast) [reply gradle/wrapper/gradle-wrapper.properties:3]: right, there are multiples additional places to update.. will take a look. thanks!

**Comment by ncordon** [comment]
@dbast, thanks for your contribution. Could you explain why if I ran the suggested command

```
./gradlew wrapper --gradle-version 6.9.2
```

on the root of the repo and the `docs` and `extra-dependencies` folders on top of your PR I get differences? Could you run the command again on top of your PR and double check, please?

Sorry to be inquisitive, but this is quite a big change even if it is automated.

**Comment by ncordon** [comment]
Sorry @dbast but it looks like the project does not work well with this changes (at least in the CI). I've tried forcing re-building everything from scratch and not using a cached version in [dec6513](https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/2962/commits/dec65138ab471d11f7862f6cf696e91a3a01710f) but it's still not working, so any ideas?

**Pull request closed without merging**